qiskit_ibm_runtime/visualization/draw_execution_spans.py

```diff
@@ -0,0 +1,115 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Functions to visualize :class:`~.ExecutionSpans` objects."""
+
+from __future__ import annotations
+
+from functools import partial
+from itertools import cycle
+from datetime import datetime, timedelta
+from typing import TYPE_CHECKING
+
+from ..execution_span import ExecutionSpan, ExecutionSpans
+from .utils import plotly_module
+
+if TYPE_CHECKING:
+    from plotly.graph_objects import Figure as PlotlyFigure
+
+
+HOVER_TEMPLATE = "<br>".join(
+    [
+        "<b>ExecutionSpans{id}[{idx}]</b>",
+        "<b>&nbsp;&nbsp;&nbsp;Start:</b> {span.start:%Y-%m-%d %H:%M:%S.%f}",
+        "<b>&nbsp;&nbsp;&nbsp;Stop:</b> {span.stop:%Y-%m-%d %H:%M:%S.%f}",
+        "<b>&nbsp;&nbsp;&nbsp;Size:</b> {span.size}",
+        "<b>&nbsp;&nbsp;&nbsp;Pub Indexes:</b> {idxs}",
+    ]
+)
+
+
+def _get_idxs(span: ExecutionSpan, limit: int = 10) -> str:
+    if len(idxs := span.pub_idxs) <= limit:
+        return str(idxs)
+    else:
+        return f"[{', '.join(map(str, idxs[:limit]))}, ...]"
+
+
+def _get_id(span: ExecutionSpan, multiple: bool) -> str:
+    return f"<{hex(id(span))}>" if multiple else ""
+
+
+def draw_execution_spans(
+    *list_of_spans: ExecutionSpans, common_start: bool = False, normalize_y: bool = False
```

Comment:
```suggestion
    *spans: ExecutionSpans, common_start: bool = False, normalize_y: bool = False
```
Non-blocking suggestion, take it or leave it

Reply:
[ef424b7](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1923/commits/ef424b7535b42b4e6be122fa1fd2f1f20cafe1f5)